content/docs/studio/user-guide/team-collaboration/index.md

```diff
@@ -7,12 +7,21 @@ team.
 
 In this page, you will learn about:
 
-- [How to create a team](#create-a-team)
-- [How to invite collaborators (team members)](#invite-collaborators)
-- [The privileges (access permissions) of different roles](#roles)
-- [How to manage the team and its projects](#manage-your-team-and-its-projects)
-- [How to manage connections to self-hosted GitLab servers](#manage-connections-to-self-hosted-gitlab-servers)
-- [Why and how to change your team plan and team size](#change-your-team-plan-and-team-size)
+- [Teams](#teams)
```

Comment:
@jellebouwman Is there a particular reason to make this change? I feel like it was better before. If you still want to go ahead with this change, can we make it into a separate PR and limit the current PR to `force import` changes?

Reply:
Sorry @tapadipti - my IDE seems to have included these changes without me noticing.